functions/exercise6/script.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+function findTheLongestWordWithinTheSentence(sentence) {
+    var words = sentence.match(/\w[a-z]{0,}/gi);
```

Comment:
Do you understand what this line does?
It works, but I have a feeling it is too complicated comparing with what we want to achieve.
But if you can explain it to me, then let's leave it. So please either change it to something simpler or write eplanation what does this line do in details.